src/main/webapp/app/exercises/shared/feedback/standalone-feedback/standalone-feedback.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit, Optional } from '@angular/core';
+import { Exercise, ExerciseType } from 'app/entities/exercise.model';
+import { Result } from 'app/entities/result.model';
+import dayjs from 'dayjs/esm';
+import { ExerciseService } from 'app/exercises/shared/exercise/exercise.service';
+import { ActivatedRoute } from '@angular/router';
+import { HttpResponse } from '@angular/common/http';
+import { ExerciseCacheService } from 'app/exercises/shared/exercise/exercise-cache.service';
+
+@Component({
+    selector: 'jhi-standalone-feedback',
+    templateUrl: './standalone-feedback.component.html',
+    styleUrls: ['./../feedback.scss'],
+})
+export class StandaloneFeedbackComponent implements OnInit {
+    exercise?: Exercise;
+    result?: Result;
+
+    isTemplateStatusMissing = false;
+    showMissingAutomaticFeedbackInformation = false;
+    messageKey?: string = undefined;
+    exerciseType?: ExerciseType = undefined;
```

Comment:
Using `?:`initializes the attribute with undefined by default, this explicit initialization is not necessary. 
```suggestion
    messageKey?: string;
    exerciseType?: ExerciseType;
```